OpenAPI/openapi3.yaml

```diff
@@ -81,6 +81,11 @@ paths:
       requestBody:
         description: (Possibly partial) content of the file. Required if Content-Length > 0.
         required: false
+        content:
+          application/offset+octet-stream":
```

Comment:
The double quote at the end of this line looks suspicious. Is this intended?

Reply:
Hello, thx for replying. No it's not intended indeed. Go figure how it got there lol.
I'll push a fix.